### DISCUSSION TITLE
Access cluster from outside

### DISCUSSION THREAD

**Discussion by jfmatth** [cloudnative-pg/cloudnative-pg#1690] [General]
Hi, Love this project.  It seems to be everything i need for my simple django app, thank you!

I'd like to expose the cluster 5432 to outside the cluster, but don't quite understand the nginx example.  I'm using K3s with Traefik and your example doesn't seem to align.

Any help would be appreciated.

**Comment by gbartolini** [comment]
Hi @jfmatth,

Thanks for using CloudNativePG and your feedback.

I haven't used Traefik so I cannot comment on that (we are focused on PostgreSQL primarily, and provide ways to integrate in Kubernetes). While we try and provide examples in the docs on how to reach the database from outside a cluster, we cannot cover all cases. This is ultimately a Kubernetes domain "problem", and you should look into ways of exposing Kubernetes services outside.

We recommend running applications together with the database, for a truly cloud native experience.

Having said this, we encourage users to add sections to the docs if they feel are relevant. So if somebody in the community wants to step in, they are more than welcome.

Cheers,
Gabriele
